.github/workflows/quality-gate.yml

```
@@ -38,8 +45,11 @@ jobs:
 
       - name: Playwright E2E tests
         run: npm run test:e2e
+        env:
+          SKIP_ENV_VALIDATION: "1"
 
       - name: Secret scan
         uses: gitleaks/gitleaks-action@v2
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        working-directory: .
```

actionlint (1.7.12)

[error] unexpected key "working-directory" for step to execute action. expected one of "continue-on-error", "env", "id", "if", "name", "timeout-minutes", "uses", "with"

(syntax-check)